dotCMS/src/main/java/com/dotmarketing/image/filter/ImageFilterApiImpl.java

```diff
@@ -1,12 +1,94 @@
 package com.dotmarketing.image.filter;
 
 import java.io.File;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class ImageFilterApiImpl implements ImageFilterAPI {
 
-	public File getImageFileFromUri(String uri) {
-		// TODO Auto-generated method stub
-		return null;
-	}
 
+    public File getImageFileFromUri(String uri) {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    public String getVariableFromPath(String path) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'path' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)